test/data/err_176.py

```diff
@@ -0,0 +1,9 @@
+from datetime import datetime, timezone
+
+# Should warn:
+start_date = datetime.utcnow()
+old_date = datetime.utcfromtimestamp(1)
```

Comment:
Add/update the test cases from the above comment:

```python
from datetime import datetime, timezone
import datetime as dt

# Should warn:
start_date = datetime.utcnow()
old_date = datetime.utcfromtimestamp(1)
start_date = dt.datetime.utcnow()
old_date = dt.datetime.utcfromtimestamp(1)
```

Make sure to run `make update-tests` to update the corresponding `txt` file.